server/utils/placements/index.ts

```diff
@@ -225,17 +219,11 @@ export const departureInformation = (placement: Cas1SpaceBooking): SummaryList =
 
 export const requirementsInformation = (placement: Cas1SpaceBooking): SummaryList => {
   const requirements = placement.characteristics
-  const apType =
-    apTypeCriteriaLabels[
-      (requirements.find(requirement => specialistApTypeCriteria.includes(requirement as SpecialistApTypeCriteria)) ||
-        'normal') as ApTypeCriteria
-    ]
   const apRequirements = filterApLevelCriteria(requirements)
   const roomRequirements = filterRoomLevelCriteria(requirements)
 
   return {
     rows: [
-      summaryListItem('AP type', apType),
```

Comment:
Is this a conscious decision by the designers, or just an accident. It makes no sense to me to show the AP characteristics, but not show that it's a PIPE AP say.
Unless they can come up with a very good reason to remove it, you should leave the AP type in.